### ISSUE TITLE
Use network storage for backups

### ISSUE THREAD

**Issue by DennisVredeveld** [sanderdw/hassio-addons#82]
Guess this more of a feature request than an issue, so feel free to close if not feasible/possible. 
It seems that only the local /backup directory is mounted in de docker container. As a result, I can only use this location for backups within DSMR reader. 
However, HA now supports network storage, which enables users to configure external location as HA backup location. The local backup folder is unused in that case. It would be nice if this network share was also available in the container, so I can use it also for DSMR backups. That way, all backups are at the same location.

**Comment by sanderdw** [comment]
Good suggestion, wil definitely have a look when there is the need for a new version.

**Comment by sanderdw** [comment]
Hi @DennisVredeveld, little bit late but you can now use remote backups in the new addon version 1.12.0, see updated readme here (step 10): https://github.com/sanderdw/hassio-addons/blob/master/dsmr_reader/README.md

**Comment by DennisVredeveld** [comment]
Thanks a lot @sanderdw  works like a charm!
Any particular reason why the remote location has to be of type Share? I was expecting type Backup (where my backups are :-)) would also be allowed.

**Comment by sanderdw** [comment]
Yeah I wanted to work like that as well but when you add a remote share as type backup it’s not mounted directly on the file system, which is required for mapping it to the addon. So HA backup shares are basically only available to the native HA backup functionality.

So therefore I chose the Share type (also better naming than the Media type).
